lte/gateway/python/integ_tests/Makefile

```diff
@@ -64,9 +68,8 @@ else
 endif
 endif
 
-.PHONY: fed_integ_test
-fed_integ_test: prepare_environment
-	$(eval export FEDERATED_MODE = True)
+.PHONY: federated_integ_test
+federated_integ_test: prepare_environment prepare_federation
```

Comment:
Could you please exchange `fed_integ_test` -> `federated_integ_test` on line 83 for consistency?